cmd/snapd/main_test.go

```diff
@@ -63,7 +63,7 @@ func (s *snapdSuite) TestSanityFailGoesIntoDegradedMode(c *C) {
 	restore = apparmor.MockIsHomeUsingNFS(func() (bool, error) { return false, nil })
 	defer restore()
 	restore = seccomp.MockSnapSeccompVersionInfo(func(s seccomp.Compiler) (string, error) {
-		return "abcdef 1.2.3 1234abcd", nil
+		return "abcdef 1.2.3 1234abcd -", nil
```

Comment:
https://twitter.com/zygoon/status/1115944564441997312 (half serious ;-)

Reply:
The format is already decided: space separated fields. Changing that isn't a big deal to me, but should be in a different PR imho.

Reply:
I just found the - funny, it is a special marker to indicate that a variable list component is absent in mountinfo. I have no problem with this.